packages/devextreme/testing/testcafe/tests/scheduler/header/dateNavigator.ts

```diff
@@ -120,6 +120,31 @@ test('Current date in Navigator should be respond on Current date of Calendar',
   height: 400,
 }));
 
+test('Current date in navigator should be updated if scheduler currentDate is changed', async (t) => {
+  const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+  const scheduler = new Scheduler('#container');
+
+  await scheduler.option('currentDate', new Date(2022, 2, 28));
+
+  await t
+    .click(scheduler.toolbar.navigator.caption);
+
+  await t
+    .expect(await takeScreenshot('navigator-state-after-change-currentDate-option.png'))
```

Comment:
Question.

Why we need the ```t.expect(...).ok()``` here?
Maybe the ```await takeScreenshot``` is enough? :)

Reply:
It was copied from upper test, will change